### ISSUE TITLE
txt2man is partially incompatible with gawk 5.0.1

### ISSUE THREAD

**Issue by eribertomota** [mvertes/txt2man#21]
Hi @mvertes,

gawk 5.0.1 was uploaded to Debian on 2019-11-03[1]. Since this date, the Debian CI failed to test txt2man[2][3]. See here a small test:

```
$ cat test.txt 
NAME
  test - do a test
DESCRIPTION
  Do a test.
$ txt2man test.txt > /dev/null
gawk: cmd. line:58: warning: regexp escape sequence `\o' is not a known regexp operator
gawk: cmd. line:116: warning: regexp escape sequence `\,' is not a known regexp operator
gawk: cmd. line:116: warning: regexp escape sequence `\;' is not a known regexp operator
```

I found here[4] a tip about how to fix this issue.

[1] https://tracker.debian.org/pkg/gawk
[2] https://ci.debian.net/packages/t/txt2man/unstable/amd64/
[3] https://ci.debian.net/data/autopkgtest/unstable/amd64/t/txt2man/3329134/log.gz
[4] https://github.com/KittyKatt/screenFetch/issues/627#issuecomment-526612183

Cheers,

Eriberto

**Comment by eribertomota** [comment]
Hum... I saw now that #18 seems solve the issue...

**Comment by mvertes** [comment]
fixed by #55
